bin/nvcc_wrapper

```diff
@@ -229,7 +229,7 @@ do
     fi
     ;;
   #Handle known nvcc args
-  --dryrun|--verbose|--keep|--source-in-ptx|-src-in-ptx|--keep-dir*|-G|-lineinfo|-extended-lambda|-expt-extended-lambda|-expt-relaxed-constexpr|--resource-usage|--fmad=*|--use_fast_math|--Wext-lambda-captures-this|-Wext-lambda-captures-this)
+  --dryrun|--verbose|--keep|--source-in-ptx|-src-in-ptx|--keep-dir*|-G|-lineinfo|--generate-line-info|-extended-lambda|-expt-extended-lambda|-expt-relaxed-constexpr|--resource-usage|--fmad=*|--use_fast_math|--Wext-lambda-captures-this|-Wext-lambda-captures-this)
```

Comment:
seems to me this is already present via the shortcut `-lineinfo`

Reply:
@fnrizzi You're right that `--generate-line-info` and `-lineinfo` are the same thing.

However, when reading the output of `nvcc --help` or reading [the documentation](https://docs.nvidia.com/cuda/pdf/CUDA_Compiler_Driver_NVCC.pdf), it seems that `--generate-line-info` is the preferred choice (since `-lineinfo` is always in parentheses).

My own opinion (for what it's worth) is that:
1. `nvcc_wrapper` should be able to allow **any** valid `nvcc` argument.
2. It tend to prefer the `--generate-line-info` which makes more sense when reading it ("generate line information"), while `-lineinfo` is slightly more cryptic...

Reply:
i think it is just an alias for it, not necessarily a preferred way. 
In any case, I agree that nvcc_wrapper should accept any valid arg. So I would suggest to modify thjis PR so that for every flag, we allow both the "long" version as well as the "short" one.

Reply:
Ideally that's what I should do... However, I must admit I'm out of time this week :timer_clock: On a very pragmatic side of the things, I guess `nvcc_wrapper` could be enriched "on the go" with users' requests. What do you think?

Reply:
ok, how about a follow up PR for this?

Reply:
Fine with me. How would I allow you to modify my branch ?

Reply:
i just posted the suggestion  below should be easy to just commit that

Reply:
Ok, just tell me when you're OK with the suggestion and I'll push it :wink:

Reply:
try now please :)

Reply:
please change the description and i am good ! thank you for the PR!